.github/workflows/scripts/run_tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -e
+
+# C++
+bazel test //psi_cardinality/cpp/...
+
+# C
+bazel test //psi_cardinality/c/...
+
+# Go
+bazel test //psi_cardinality/go/...
+
+# JavaScript
+npm run submodule:update
+npm run em:update
+npm run em:init
+npm install
+npm run build
```

Comment:
This will call the `-c opt` internally so it will not use the previously compiled bazel cache.

Reply:
Turns out it will always rebuild because bazel --cpu has changed.